### PR TITLE
Add com.github.Flacon

### PR DESCRIPTION
**Pull request by Eonfge**
Packaged version of Flacon:
https://github.com/flacon/flacon

Upsteam development actually maintains its own Flatpak bundle, so adding it to the Flathub repository is very easy.

### CODE REVIEW

**Comment by Eonfge** [comment]
bot, build com.github.Flacon

**Comment by Eonfge** [comment]
bot, build com.github.Flacon

**Comment by Eonfge** [comment]
bot, build com.github.Flacon

**Comment by bilelmoussaoui** [comment]
The app-id is wrong. It should be com.github.flacon.Falcon and you should probably discuss that with upstream

File: com.github.Flacon.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Kevin Degeling -->
+
+<component type="desktop">
```
Comment (bilelmoussaoui) [inline com.github.Flacon.appdata.xml]:
```suggestion
<component type="desktop-application">
```

File: com.github.Flacon.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline com.github.Flacon.appdata.xml]: The file should be upstreamed and installed by project's build system.

**Comment by barthalion** [comment]
FYI @SokoloffA

**Comment by Eonfge** [comment]
@barthalion https://github.com/flacon/flacon/issues/97 I've mentioned it to @sokoloffA but he didn't show any interest in including it on Flathub. Since I was already running it from my own repository for a few weeks, I decided to just submit my manifest.

File: com.github.Flacon.appdata.xml (hunk 1 above)
Comment (Eonfge) [reply com.github.Flacon.appdata.xml]: Corrected

File: com.github.Flacon.appdata.xml (hunk 2 above)
Comment (Eonfge) [reply com.github.Flacon.appdata.xml]: I've added it as a feature request:
https://github.com/flacon/flacon/issues/101

I've also added a note to the file

**Comment by Eonfge** [comment]
bot, build com.github.Flacon

**Comment by Eonfge** [comment]
> com.github.flacon.Falcon

@bilelmoussaoui  Upstream chose `com.github.Flacon` and I discussed the issue here: https://github.com/flacon/flacon/issues/97

You think I should just cancel this pull request and wait for upstream to do it instead? I was under the impression that upstream didn't want to add it's package to Flathub, so I decided to do that.

File: com.github.Flacon.appdata.xml (hunk 2 above)
Comment (Eonfge) [reply com.github.Flacon.appdata.xml]: Issue resolved upstream. Should be included with v5.4.1

**Comment by Eonfge** [comment]
@bilelmoussaoui & @barthalion in discussing an upstream patch to the appdata.xml, @SokoloffA [responded positively to my Flathub submission](https://github.com/flacon/flacon/issues/101#issuecomment-535657571). He might not feel much for getting hands on, but he is certainly supportive to me submitting it.

File: com.github.Flacon.yaml [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+sdk: org.kde.Sdk
+command: flacon
+rename-desktop-file: flacon.desktop
+rename-icon: flacon
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=x11
+  - --share=ipc
+
```
Comment (bilelmoussaoui) [inline com.github.Flacon.yaml]: You need to give it access to device=dri as I'm getting this when I run it 
```
libEGL warning: wayland-egl: could not open /dev/dri/card0 (No such file or directory)
```

Comment (Eonfge) [reply com.github.Flacon.yaml]: Added

**Comment by Eonfge** [comment]
bot, build com.github.Flacon

**Comment by bilelmoussaoui** [comment]
Repository has been created: https://github.com/flathub/com.github.Flacon


Thanks!

**Comment by bilelmoussaoui** [comment]
@Eonfge @SokoloffA Invitations for write-access to the repository were sent. Thanks!

**Pull request closed without merging**